readme.md

```diff
@@ -174,10 +190,41 @@ console.log(chalk.styles.red.open + 'Hello' + chalk.styles.red.close);
 - `bgWhite`
 
 
-## 256-colors
+## 256/16 million (Truecolor) color support
+
+Chalk supports 256 colors and, when manually specified, [Truecolor (16 million colors)](https://gist.github.com/XVilka/8346728) on all supported terminal emulators.
+
+For supported methods (listed below), the color will be 'fit' to the color level supported (i.e. RGB colors will be downsampled to 16 colors if only basic support is enabled).
```

Comment:
I think `downsampled` was the word I was looking for here.

E.g. if your support level is 2 (256 colors) and you give it an RGB, it will downsample the full 16 million color down to the 256-palette color.

Is `downsample` correct here? I want to be a bit more specific than `convert` since in this context it could be mistaken as converting to another color model (which is technically true, I suppose...)